faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -1,12 +1,14 @@
 package faunadb.values
 
-import com.fasterxml.jackson.annotation.{ JsonIgnore, JsonProperty, JsonValue }
```

Comment:
We should use core's style here:
- Organize import alphabetically
- Have spaces within the braces